apple/internal/resource_rules/BUILD

```diff
@@ -100,6 +100,17 @@ bzl_library(
     ],
 )
 
+bzl_library(
+    name = "apple_precompiled_resource_bundle",
+    srcs = ["apple_precompiled_resource_bundle.bzl"],
+    visibility = [
+        "//apple:__subpackages__",
+    ],
+    deps = [
+        "//apple/internal:providers",
```

Comment:
This should list a lot more,  like `bazel_skylib`, `apple_toolchains`, `rule_support`, etc.